buildenv/jenkins/docker-slaves/jenkinsfile-build-container

```diff
@@ -0,0 +1,80 @@
+/*******************************************************************************
+ * Copyright (c) 2018, 2018 IBM Corp. and others
+ *
+ * This program and the accompanying materials are made available 
+under
+ * the terms of the Eclipse Public License 2.0 which accompanies 
+this
+ * distribution and is available at 
+https://www.eclipse.org/legal/epl-2.0/
+ * or the Apache License, Version 2.0 which accompanies this 
+distribution and
+ * is available at https://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * This Source Code may also be made available under the following
+ * Secondary Licenses when the conditions for such availability set
+ * forth in the Eclipse Public License, v. 2.0 are satisfied: GNU
+ * General Public License, version 2 with the GNU Classpath
+ * Exception [1] and GNU General Public License, version 2 with the
+ * OpenJDK Assembly Exception [2].
+ *
+ * [1] https://www.gnu.org/software/classpath/license.html
+ * [2] http://openjdk.java.net/legal/assembly-exception.html
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH 
+Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH 
+Assembly-exception
+ 
+*******************************************************************************/
+
+if(params.ghprbPullId) {
+    PARSE = "${ghprbCommentBody}".split()
+    for(i = 0; i < PARSE.length - 4; i++) {
+        if(PARSE[i] == "Jenkins" && PARSE[i+1] == "build" && PARSE[i+2] == "docker") {
+            ARCH = PARSE[i+3]
+            OS = PARSE[i+4]
+        }
+    }
+}
+
+NAMESPACE = "eclipse"
+REPOSITORY = "${NAMESPACE}/openj9-jenkins-agent-${ARCH}-${OS}"
+timeout(time: 3, unit: 'HOURS') {
+    timestamps {
+        node("sw.tool.docker&&hw.arch.${ARCH}") {
```

Comment:
as well as the node we hit.
see https://github.com/eclipse/openj9/blob/master/buildenv/jenkins/common/build#L334